flow_action_components/GetOpportunityPartners/force-app/main/default/classes/GetOpportunityPartnersAction.cls

```diff
@@ -0,0 +1,132 @@
+/**
+* @author       : Tamar Erlich
+* @date         : October 07, 2020
+* @description  : Invocable method that given opportunityId and accountId, returns a list of opportunity partners to display on opportunity plan record page
+* Note          : This class is called by the Flow.
+* *************************************************
+*       <runTest>GetOpportunityPartnersActionTest<runTest>
+* *************************************************
+* @History
+* -------
+* VERSION | AUTHOR                | DATE               | DESCRIPTION
+* 1.0     | Tamar Erlich          | October 07, 2020   | Created
+**/
+global with sharing class GetOpportunityPartnersAction {
+
+    // Attributes passed in from the Flow
+    global class Requests {
```

Comment:
If you look at the other actions, you'll see that our convention is to start with the invocable method and put the Request and Response classes at the bottom. It's not necessary, but the consistency makes things a little easier to work with.